blocks/library/gallery/index.js

```diff
@@ -61,16 +70,11 @@ registerBlockType( 'core/gallery', {
 			query( 'div.blocks-gallery figure.blocks-gallery-image img', {
 				url: attr( 'src' ),
 				alt: attr( 'alt' ),
-			} ),
+			} ) || [],
```

Comment:
@aduth if others’ code isn't part of a webpack build how would they import `registerBlockType `, so that they have access to the API? And if  they’re already using webpack, ES6 becomes a reasonable expectation.